packages/direction-links/tests/site/index.html

```diff
@@ -48,138 +48,138 @@ <h1>Test: direction-links</h1>
+			<button class="au-direction-link">Skip to footer<span class="au-direction-link--down" aria-hidden="true"></span></button>
 		</div>
 
 		<div class="split au-body split--alt split--dark">
 
 			<h2>bare <code>--dark</code></h2>
-			<a class="au-direction-link au-direction-link--dark" href="#">Continue</a>
+			<a class="au-direction-link au-direction-link--dark" href="#">Continue<span class="au-direction-link--dark au-direction-link--right" aria-hidden="true"></span></a>
 
 			<hr>
 			<h2><code>--right</code> <code>--dark</code> variant</h2>
 
-			<a class="au-direction-link au-direction-link--right au-direction-link--dark" href="#">Next</a>
+			<a class="au-direction-link au-direction-link--dark" href="#">Next<span class="au-direction-link--dark au-direction-link--right" aria-hidden="true"></span></a>
 
 			<hr>
 			<h2><code>--left</code> <code>--dark</code> variant</h2>
 
-			<a class="au-direction-link au-direction-link--left au-direction-link--dark" href="#">Back</a>
+			<a class="au-direction-link au-direction-link--dark" href="#">Back<span class="au-direction-link--dark au-direction-link--left" aria-hidden="true"></span></a>
 
 			<hr>
 			<h2><code>--up</code> <code>--dark</code> variant</h2>
 
-			<a class="au-direction-link au-direction-link--up au-direction-link--dark" href="#">Top</a>
+			<a class="au-direction-link au-direction-link--dark" href="#">Top<span class="au-direction-link--dark au-direction-link--up" aria-hidden="true"></span></a>
 
 			<hr>
 			<h2><code>--down</code> <code>--dark</code> variant</h2>
 
-			<a class="au-direction-link au-direction-link--down au-direction-link--dark" href="#">Skip to footer</a>
+			<a class="au-direction-link au-direction-link--dark" href="#">Skip to footer<span class="au-direction-link--dark au-direction-link--down" aria-hidden="true"></span></a>
 
 			<hr>
 			<h2>direction-links as button <code>--dark</code></h2>
 
-			<button class="au-direction-link au-direction-link--right au-direction-link--dark">Next</button>
-			<button class="au-direction-link au-direction-link--left au-direction-link--dark">Back</button>
-			<button class="au-direction-link au-direction-link--up au-direction-link--dark">Top</button>
-			<button class="au-direction-link au-direction-link--down au-direction-link--dark">Skip to footer</button>
+			<button class="au-direction-link au-direction-link--dark">Next<span class="au-direction-link--dark au-direction-link--right" aria-hidden="true"></span></button>
+			<button class="au-direction-link au-direction-link--dark">Back<span class="au-direction-link--dark au-direction-link--left" aria-hidden="true"></span></button>
+			<button class="au-direction-link au-direction-link--dark">Top<span class="au-direction-link--dark au-direction-link--up" aria-hidden="true"></span></button>
+			<button class="au-direction-link au-direction-link--dark">Skip to footer<span class="au-direction-link--dark  au-direction-link--down" aria-hidden="true"></span></button>
```

Comment:
do we need to have `direction-link--dark` on the `span` as well? lets try and make use of BEM